TradeDis/backend
# feat: report posts

**What was done?**
Task: TRAD-12 "As a user, I would like to be able to report an inappropriate post."
- Added `reporters` field to `Post` model
- Updated `Post` documents manually in MongoDB to ensure all documents contained the `reporters` field

**Areas which need special reviewing**
N/A

**Testing**
This test ensures that the `reporters` field is being return in the `Post` object results.
1. Ensure the backend server is running (`yarn dev` in the backend project)
2. Request the endpoint `/api/v1/posts`
3. Verify that the returned posts include the `reporters` field
